observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,88 @@ The Network Observability Operator enables administrators to observe and analyze
 These release notes track the development of the Network Observability Operator in the {product-title}.
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+
+[id="network-observability-operator-release-notes-1-8_{context}"]
+== Network Observability Operator 1.8.0
+The following advisory is available for the Network Observability Operator 1.8.0:
+
+* link:https://access.redhat.com/errata/RHSA-2024:8014[Network Observability Operator 1.8.0]
+
+[id="network-observability-operator-1.8.0-features-enhancements_{context}"]
+=== New features and enhancements
+
+[id="network-observability-operator-pkt-xlat-1-8_{context}"]
+==== Packet translation
+You can now enrich network flows with translated endpoint information, showing not only the service but also the specific backend pod, so you can see which pod served a request.
+
+[id="network-observability-operator-OVN-observability-1-8_{context}"]
+==== OVN-Kubernetes networking events tracking
+
+:FeatureName: OVN-Kubernetes networking events tracking
+include::snippets/technology-preview.adoc[]
+
+You can now use network event tracking in Network Observability to gain insight into OVN-Kubernetes events, including network policies, admin network policies, and egress firewalls.
+
+[id="network-observability-operator-ebpf-perf-improvements-1-8_{context}"]
+==== eBPF performance improvements in 1.8
+
+* Network Observability now uses hash maps instead of per-CPU maps. This means that network flows data is now tracked in the kernel space and new packets are also aggregated there. The de-duplication of network flows can now occur in the kernel, so the size of data transfer between the kernel and the user spaces yields better performance. With these eBPF performance improvements, there is potential to observe a CPU resource reduction between 40% and 57% in the eBPF Agent.
+
+[id="network-observability-cli-1-8_{context}"]
+==== Network Observability CLI
+The following new features, options, and filters are added to the Network Observability CLI for this release:
+
+* Capture metrics with filters enabled by running the `oc netobserv metrics` command.
+* Run the CLI in the background by using the `--background` option with flows and packets capture and running `oc netobserv follow` to see the progress of the background run and `oc netobserv copy` to download the generated logs.
+* Enrich flows and metrics capture with Machines, Pods, and Services subnets by using the `--get-subnets` option.
+* New filtering options available with packets, flows, and metrics capture:
+
+** eBPF filters on IPs, Ports, Protocol, Action, TCP Flags and more
```

Comment:
FYI @jpinsonneau added eBPF filters per your comment in the other RN PR.

Reply:
Perfect ! thanks @skrthomas